docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,242 @@
+  `EXPLAIN ANALYZE VERBOSE`. ({issue}`15286`)
+* Add stage skewness warnings to the output of `EXPLAIN ANALYZE`. ({issue}`15286`)
+* Add support for `ALTER COLUMN ... SET DATA TYPE` statement. ({issue}`11608`)
+* Allow configuring a refresh interval for the DB resource group manager with
+  the `resource-groups.refresh-interval` configuration property. ({issue}`14514`)
+* Improve performance of queries that compare `date` columns with
+  `timestamp(n) with time zone` literals. ({issue}`5798`)
+* Improve performance and resource utilization when inserting into tables. ({issue}`14718`, {issue}`14874`)
+* Improve performance for `INSERT` queries when fault-tolerant execution is
+  enabled. ({issue}`14735`)
+* Improve planning performance for queries with many `GROUP BY` clauses. ({issue}`15292`)
+* Improve query performance for large clusters and skewed queries. ({issue}`15369`)
+* Rename the `node-scheduler.max-pending-splits-per-task` configuration property
+  to `node-scheduler.min-pending-splits-per-task`. ({issue}`15168`)
+* Ensure that the configured number of task retries is not larger than 126. ({issue}`14459`)
+* Fix incorrect rounding of `time(n)` and `time(n) with time zone` values near
+  the top of the range of allowed values. ({issue}`15138`)
+* Fix incorrect results for queries involving window functions without a
+  `PARTITION BY` clause followed by the evaluation of window functions with a
+  `PARTITION BY` and `ORDER BY` clause. ({issue}`15203`)
+* Fix incorrect results when adding or subtracting an `interval` from a
+  `timestamp with time zone`. ({issue}`15103`)
+* Fix potential incorrect results when joining tables on indexed and non-indexed
+  columns at the same time. ({issue}`15334`)
+* Fix potential failure of queries involving `MATCH_RECOGNIZE`. ({issue}`15343`)
+* Fix incorrect reporting of `Projection CPU time` in the output of `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`15364`)
+* Fix `SET TIME ZONE LOCAL` to correctly reset to the initial time zone of the
+  client session. ({issue}`15314`)
+
+## Security
+
+* Add support for string replacement as part of
+  [impersonation rules](system-file-auth-impersonation-rules). ({issue}`14962`)
+* Add support for fetching access control rules via HTTPS URLs. ({issue}`14008`)
+* Fix some `system.metadata` tables improperly showing the names of catalogs
+  which the user cannot access. ({issue}`14000`)
+* Fix `USE` statement improperly disclosing the names of catalogs and schemas
+  which the user cannot access. ({issue}`14208`)
+* Fix improper HTTP redirect after OAuth 2.0 token refresh. ({issue}`15336`)
+
+## Web UI
+
+* Display operator CPU time in the "Stage Performance" tab. ({issue}`15339`)
+
+## JDBC driver
+
+* Return correct values in `NULLABLE` columns of the
+  `DatabaseMetaData.getColumns` result. ({issue}`15214`)
+
+## BigQuery connector
+
+* Add experimental support for Arrow serialization when reading from BigQuery.
```

Comment:
Do we need to call it Apache Arrow ? And link? Or mention what it is?

Reply:
Also ... what is the user facing result of doing that? What is it for?

Reply:
For better read performance.